cmd/api/src/api/v2/auth/sso.go

```diff
@@ -122,3 +123,24 @@ func (s ManagementResource) ListAuthProviders(response http.ResponseWriter, requ
 		}
 	}
 }
+
+func (s ManagementResource) SSOLoginHandler(response http.ResponseWriter, request *http.Request) {
+	ssoProviderSlug := mux.Vars(request)[api.URIPathVariableSSOProviderSlug]
+
+	if ssoProvider, err := s.db.GetSSOProviderBySlug(request.Context(), ssoProviderSlug); err != nil {
+		api.HandleDatabaseError(request, response, err)
+	} else {
+		switch ssoProvider.Type {
+		case model.SessionAuthProviderSAML:
+			//todo handle saml login
+		case model.SessionAuthProviderOIDC:
+			if oidcProvider, err := s.db.GetOIDCProviderBySSOProviderID(request.Context(), ssoProvider.ID); err != nil {
+				api.HandleDatabaseError(request, response, err)
+			} else {
+				s.OIDCLoginHandler(response, request, ssoProvider, oidcProvider)
+			}
+		default:
+			return
```

Comment:
Edge case: Consider returning `501 - Not implemented`. This will cover the case where clearly we've found registered sso provider but don't have a login handler to support it.